guide/14-deep-learning/how_estimate_batch_size_works.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<p>Prepare the data for the <code>SingleShotDetector</code> class using <code>prepare_data()</code> in <code>arcgis.learn</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1533/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done</p>